### PR TITLE
Javascript timers

### PR DESCRIPTION
**Pull request by annevillarete**
Untitled_ Mar 13, 2021 11_57 AM

### CODE REVIEW

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]
@annevillarete Your `javascript-bank` code-exercise solution should be on its own branch in its own pull request.

File: javascript-timers/timeout.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+function timeout() {
+    setTimeout(function () {
+        var $h1 = document.querySelector("h1")
+        $h1.innerHTML = 'Hello There'
```
Comment (TimelikeClosure) [inline javascript-timers/timeout.js]: Using `innerHTML` is dangerous and slow. Please use `textContent` instead.

File: javascript-timers/interval.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+function countdown() {
+    var $h1 = document.querySelector("h1");
+    var i = 4;
+    var timerID = setInterval(function () {
+        $h1.innerHTML = i;
+        i -= 1;
+        if (i < 0) {
+            clearInterval(timerID);
+            $h1.innerHTML = '~Earth Beeeelooowww Us~';
```
Comment (TimelikeClosure) [inline javascript-timers/interval.js]: Same as above.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: javascript-timers/timeout.js (hunk 1 above)
Comment (annevillarete) [reply javascript-timers/timeout.js]: This has been updated.

File: javascript-timers/interval.js (hunk 2 above)
Comment (annevillarete) [reply javascript-timers/interval.js]: This has also been updated.

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**